src/public/less/styles.less

```diff
@@ -141,10 +140,22 @@ body {
     left: 25%;
 }
 
+.pages {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    min-height: 100vh;
+}
```

Comment:
all our content is wrapped in .pages which is a flex container. the `justify-content` and `min-height` rules ensure that the footer hugs the bottom of the window.